platforms/nuttx/cmake/px4_impl_os.cmake

```diff
@@ -72,6 +72,8 @@ function(px4_os_add_flags)
 	# prevent using the toolchain's std c++ library
 	add_compile_options($<$<COMPILE_LANGUAGE:CXX>:-nostdinc++>)
 
+	add_compile_options(-mno-unaligned-access)
```

Comment:
Why did you add this?

Reply:
Considering other possible alignment issues. Removing from this PR.